lottie/src/main/java/com/airbnb/lottie/parser/AsyncCompositionLoader.java

```diff
@@ -21,7 +22,8 @@
     try {
       return LottieComposition.Factory.fromJsonSync(params[0]);
     } catch (IOException e) {
-      throw new IllegalStateException(e);
+      L.error("", e);
+      return null;
```

Comment:
the comment of `void onCompositionLoaded(@Nullable LottieComposition composition);` say _Composition will be null if there was an error loading it. Check logcat for more details._ I fix it according to this comment.

Reply:
This is a breaking API change. Please revert it from this PR. I have it on my roadmap to improve the error handling for loading.

Reply:
ok, I will revert it.